deployment/observability/grafana/dashboards/contract-event-indexer.json

```
@@ -0,0 +1,192 @@
+          "expr": "contract_event_indexer_lag_ledgers",
+          "legendFormat": "Lag (ledgers)",
+          "refId": "A"
+        }
+      ]
+    },
+    {
+      "datasource": { "type": "prometheus", "uid": "prometheus" },
+      "description": "Rate of indexer errors per second over the selected interval. A sustained non-zero rate indicates the indexer is failing to process events.",
+      "fieldConfig": {
+        "defaults": {
+          "color": { "mode": "palette-classic" },
```

Biome (2.5.3)

[error] The key color was already declared.

(lint/suspicious/noDuplicateObjectKeys)